modules/electromagnetics/test/tests/postprocessors/reflection_coefficient/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  [dim_error]
```

Comment:
is this object tested for regression elsewhere?

Reply:
Yes, it's used in the slab reflection benchmark, which it was initially developed for. Didn't feel appropriate to me to put this very specific error check in with those tests, since its testing something outside the bounds of the benchmark.